CRM/Utils/SQL.php

```diff
@@ -122,4 +122,17 @@ public static function isGroupByModeInDefault() {
     return TRUE;
   }
 
+  /**
+   * Is the Database set up to handle acceents.
+   * @return bool
+   */
+  public static function supportStorageOfAccents() {
+    $charSetDB = CRM_Core_DAO::executeQuery("SHOW VARIABLES LIKE 'character_set_database'")->fetchAll();
```

Comment:
I think the intent of the `testInternationalStrings()` was to raise a flag if one can't store accents. If this condition is really a prerequisite for storing accents, then we should generate a `notice` in`CRM_Utils_Check_Component_Env`? 

@seamuslee001 @mlutfy @nganivet -- see followup #12063.